public/index.html

```diff
@@ -45,6 +45,60 @@ <h3 class="oxford location-heading">Oxford</h3>
             </div>
           </div>
 
+          <div class="job">
+            <p><a class="job-title">Paid Web Developer Internship</a></p>
+            <div class="job-description">
+              <p>
+                We are excited to offer a paid 3-month, web developer internship at the Zooniverse.
+                The candidate would ideally be able to travel to office in Oxford (UK) but we won't
+                rule out a remote candidate. The internship will run over summer 2017 with the option
+                for flexible start and end dates.
+              </p>
+              <p>
+                As an intern you will work on a variety of projects, according to our current needs
+                as well as the interests and skills of the successful candidate.
+
+                Projects will involve a variety of work including but not limited to:
+                <ul>
```

Comment:
This line is a little unclear, may just be simpler to say;

'Projects may include;'  

or 

'Projects will involve a variety of tasks including (but not limited to);'